services/movie.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+
+import init_django_orm  # noqa: F401
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list | None = None,
+               actors_ids: list | None = None) -> QuerySet[Movie]:
+    query = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        query = query.filter(genres__id__in=genres_ids,
+                             actors__id__in=actors_ids)
```

Comment:
There is no need for this if condition, remove it and replace elif with if on the next lines. Remember that querysets are lazy in Django